.github/workflows/codeql-analysis.yml

```diff
@@ -9,6 +9,11 @@ on:
   schedule:
     - cron: '0 13 * * 1'
 
+permissions:
+  contents: read
+  security-events: write
+  pull-requests: read
```

Comment:
Hmm, maybe we should specify the permissions within each `job` entry instead of globally...

Reply:
it doesn't matter, as we have only one job